docs/source/contributing/writing-docs.adoc

```diff
@@ -104,7 +103,13 @@ $ make serve_docs
 ----
 
 The `make serve_docs` command builds and generates a live-preview of the documentation.
-You can access the documentation by browsing to *http://<IP-of-Docker-daemon>:4567*.
+You can access the documentation by browsing to *\http://<IP-of-Docker-daemon>:4567*.
```

Comment:
Not sure what you mean, but asciidoctor does not require the use of 'link:'. It will try to turn any text of the form http(s)://... into an URL. In this case, we don't want this to be a clickable link. For that reason one needs to use the backslash to escape the URL processing.